trunk/source/Library-Automata/src/de/uni_freiburg/informatik/ultimate/automata/nestedword/operations/IsEmptyHeuristic.java

```diff
@@ -128,15 +148,15 @@ public IsEmptyHeuristic(final AutomataLibraryServices services, final INestedWor
 			final Set<STATE> startStates, final Predicate<STATE> funIsForbiddenState,
 			final Predicate<STATE> funIsGoalState, final IHeuristic<STATE, LETTER> heuristic)
 			throws AutomataOperationCanceledException {
-		this(services, (INwaOutgoingLetterAndTransitionProvider<LETTER, STATE>) operand, startStates,
-				funIsForbiddenState, funIsGoalState, heuristic);
+		this(services, operand, startStates, funIsForbiddenState, funIsGoalState, heuristic, new ArrayList<>());
```

Comment:
minor: use `List.of()` or `Collections.emptyList()` to make it clear the list is unmodifiable